samples/Shell/Fabimals/Fabimals/Components/AnimalDetailsPage.fs

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018-2019 Fabulous contributors. See LICENSE.md for license.
+namespace Fabimals.Components
+
+open Fabulous.DynamicViews
+open Xamarin.Forms
+open Fabimals.Models
+
+module AnimalDetails =
+    type Msg = NoOp
+    type CmdMsg = NoOp
+    type ExternalMsg = NoOp
+    
+    type Model =
```

Comment:
The Model is the `Animal` Model. 
Couldn't it be reused?

Reply:
I think it's a best practice to keep it separated, even if it means duplicating the type.
`Animal` is a data type, `Model` is a state type.

Reply:
That said, maybe changing `Model` to contain an `Animal` would be better than duplicating all the fields?

Reply:
> That said, maybe changing Model to contain an Animal would be better than duplicating all the fields?

That would be better, yes. I think this way it is a bit cleaner.

Reply:
Fixed too